lib/iris/coords.py

```diff
@@ -2485,6 +2485,54 @@ def __init__(
         Create a 1D, numeric, and strictly monotonic :class:`Coord` with
         read-only points and bounds.
 
+        Args:
+
+        * points:
+            1D numpy array-like of values (or single value in the case of a
```

Comment:
Given that it’s specifically 1D, perhaps simply “Sequence of...” would be better?

Reply:
I think it would be wiser to stick with "numpy array-like", because I think that has come to be recognised as a common usage meaning "I can do an as_array() on it". 
The idea may extend to other array types like CUDA or dask, as well as numpy.
In addition to iterables (of iterables), it can mean an object that provides `.shape` + `.dtype`, or `__array_interface__` or `__array__`.
There's a nice discussion here : https://stackoverflow.com/a/40380014